Source/stores.cpp

```diff
@@ -2214,9 +2214,8 @@ void InitStores()
 
 void SetupTownStores()
 {
-	SetRndSeed(glSeedTbl[currlevel] * SDL_GetTicks());
-
 	auto &myPlayer = Players[MyPlayerId];
+	SetRndSeed(glSeedTbl[currlevel] * (myPlayer.townSeed + 1));

```

Comment:
why the +1 :)

Reply:
to still squeeze something useful out of old saves that will have this value at 0 :)